.github/workflows/legacy-continuous-integration.yml

```
@@ -0,0 +1,47 @@
+
+    strategy:
+      matrix:
+        operating-system:
+          - ubuntu-latest
+          - windows-latest
+        php-version:
+          - '7.0'
+          - '7.1'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)